### PR TITLE
Convert string to date

### PR DESCRIPTION
**Pull request by SaulCM**
Added MyDate class to convert a String to date according to specification, and added its unit tests.

### CODE REVIEW

File: src/main/java/utils/MyDate.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+     * @return boolean
+     */
+    private boolean isSimpleDate(final String simpleDate) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_SIMPLE_FORMAT);
+        try {
+            LocalDate localDate = LocalDate.parse(simpleDate, formatter);
+            ZoneId defaultZoneId = ZoneId.systemDefault();
+            this.parsedDate = Date.from(localDate.atStartOfDay(defaultZoneId).toInstant());
+            return true;
+        } catch (Exception e) {
```
Comment (JessickaMoya) [inline src/main/java/utils/MyDate.java:72]: You should do something with the exception

File: src/test/java/unit/tests/utils/MyDateTests.java [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+                {"0 months from now"},
+                {"1 hours from now"},
+                {"2 hour ago"}
+        };
+    }
+
+    @Test
+    public void getTodayShouldReturnTheCorrectDate() throws Exception {
+        MyDate myDate = new MyDate();
+        Calendar c;
```
Comment (JessickaMoya) [inline src/test/java/unit/tests/utils/MyDateTests.java:55]: Change your variable name "c" for another name that is more descriptive

File: src/main/java/utils/MyDate.java (hunk 1 above)
Comment (SaulCM) [reply src/main/java/utils/MyDate.java:72]: Returning an exception in a higher method.

File: src/test/java/unit/tests/utils/MyDateTests.java (hunk 2 above)
Comment (SaulCM) [reply src/test/java/unit/tests/utils/MyDateTests.java:55]: Ty will be fixed.

**Pull request merged**